src/controls/calendar/hooks/useWeekViewStyles.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
```

Comment:
For this one and other files - I would highly encourage you to NOT disable lint rules.